docs:Update docs ippool-affinity-node.md (#673)
spidernet-io/spiderpool: docs/usage/ippool-affinity-node.md

# SpiderIPPool node affinity

> *Spiderpool supports configuring the affinity between IP pools and Nodes, which enables only Pods running on specific Nodes to use the IP pools that be affinity to these Nodes. Node affinity should be regarded as a **filtering mechanism** rather than a [pool selection rule](TODO).*

Reviewer: ```suggestion
*Spiderpool supports for configuring the affinity between IP pools and Nodes. It means a pod running on a specific node can only use the IP pools that you have configured the affinity to the node. Node affinity should be regarded as a **filtering mechanism** rather than a [pool selection rule](TODO).*
```

Author: 海峰老师，这个 PR 其实还没写完，Labels 里带了 `not-ready` 的 PR 其实不太需要提前 review。上面这段引言其实是全部不要的，我只是随便拷了段以前不要的东西，先填着充个数。😂

Author: 并不是 "运行在特定 Node 的 Pod 只能使用 xxx IP 池" 而是 ”仅有运行在特定 Node 的 Pod 才能使用 xxx IP 池“，其实那些 Node 上的 Pod 还可以使用其他 IP 池的。

Reviewer: ok :) 主要是没看懂这句话，哈哈 😄

## Setup Spiderpool

If you have not set up Spiderpool yet, follow the guide [Quick Installation](https://github.com/spidernet-io/spiderpool/blob/main/docs/usage/install.md) for instructions on how to install and simply configure Spiderpool.

## Get started

> Since my cluster has only two Nodes (1 master and 1 worker), I have to remove the relevant taints on master Node through `kubectl taint`, so that ordinary Pods can also be scheduled to it. If your cluster has two or more worker Nodes, please ignore the step above.

Reviewer: ```suggestion
Since the cluster in this example has only two nodes (1 master and 1 worker), it is required to remove the relevant taints on the master node through `kubectl taint`, so that ordinary pods can also be scheduled to it. If your cluster has two or more worker nodes, please ignore the step above.
```
Normally, we need not to capitalize Pod and Node unless it's special in a code snippet.

Author: 其实一般 K8s 的资源在文档中都应该使用 Kind 名称（参见 K8s 官方文档，均不会全小写资源）。上文中的 "Node" 替换为 "node" 其实还说的过去，其含义就由 "资源 Node" 变成了一般语言中的 "节点"，而将 "Pods" 改掉其实就不太好了。

Reviewer: 嗯嗯，如果是引用代码中的内容，可以加个 `Pods`
